config.go

```diff
@@ -257,12 +263,30 @@ func SetupDaemonConfig(logger *logrus.Logger, configFile string) (DaemonConfig,
 		}
 	}
 
+	// Log config
+	setter.SetDefault(&logConfig.LogFormat, os.Getenv("GUBER_LOG_FORMAT"))
+	if logConfig.LogFormat != "" {
+		switch logConfig.LogFormat {
+		case "json":
+			logger.SetFormatter(&logrus.JSONFormatter{})
+		case "text":
+			logger.SetFormatter(&logrus.TextFormatter{})
+		default:
+			return conf, errors.New("GUBER_LOG_FORMAT is invalid; expected value is either json or text")
+		}
+	}
+
 	setter.SetDefault(&DebugEnabled, getEnvBool(log, "GUBER_DEBUG"))
 	if DebugEnabled {
 		logger.SetLevel(logrus.DebugLevel)
 		log.Debug("Debug enabled")
 	}
 
+	setter.SetDefault(&logConfig.LogLevel, uint32(getEnvInteger(log, "GUBER_LOG_LEVEL")))
+	if !DebugEnabled {
+		logger.SetLevel(logrus.Level(logConfig.LogLevel))
```

Comment:
Traditionally at Mailgun, we use log level strings.  That way, we can use constants like `logrus.DebugLevel` or "DEBUG" in config files.  For this, use data type `logrus.Level`.

Reply:
@Baliedge I just refactored that, could you please have a look again?

Reply:
I was thinking more along the lines of:
```go
	setter.SetDefault(&DebugEnabled, getEnvBool(log, "GUBER_DEBUG"))
	setter.SetDefault(&logLevel, os.Getenv("GUBER_LOG_LEVEL"))
	if DebugEnabled {
		logger.SetLevel(logrus.DebugLevel)
		log.Debug("Debug enabled")
	} else if logLevel != "" {
		logrusLogLevel, err := logrus.ParseLevel(logLevel)
		if err != nil {
			return conf, errors.Wrap(err, "invalid log level")
		}

		logger.SetLevel(logrusLogLevel)
	}
```

Reply:
@Baliedge exactly, I would have done it the same, it should be Ok now.